### DISCUSSION TITLE
Is there a ARM 64 Mandrel image available?

### DISCUSSION THREAD

**Discussion by gastaldi** [graalvm/mandrel#356] [Q&A] [answered]
Is there a Mandrel image for Apple Silicon M1 machines? The AMD64 shows the following warning:

```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested 

```

**Comment by Karm** [comment]
There is neither darwin amd64 nor darwin aarch64 (M1) Mandrel available.

**Comment by Karm** [comment]
There are also no immediate plans to have one.

**Comment by jerboaa** [comment, marked as answer]
We are mostly following upstream GraalVM on this. It's not yet supported there. Feel free to follow:
https://github.com/oracle/graal/issues/2666
